src/ProgressOnderwijsUtils/XmlCompression.cs

```diff
@@ -100,7 +99,7 @@ public static byte[] ToCompressedUtf8([NotNull] XDocument doc, byte[] dictionary
         /// Loads an XDocument that was saved with 'SaveUsingDeflateWithDictionary'.  You must provide the same dictionary used during compression.
         /// </summary>
         [NotNull]
-        public static XDocument FromCompressedUtf8(byte[] compressedBytes, byte[] dictionary)
+        public static XDocument FromCompressedUtf8(byte[] compressedBytes, byte[]? dictionary)

```

Comment:
dit ziet er niet uit alsof null een zinvolle waarde is, waarom de ?

Reply:
ZlibDecompressWithDictionary accepteert daar null.